docs(seqera-ai): align Enterprise Co-Scientist docs for 26.1 (#1381)
seqeralabs/docs: platform-enterprise_docs/seqera-ai/use-cases.md

---
title: "Use cases"

> description: "Learn how to use Co-Scientist CLI for bioinformatics workflows, pipeline development, and data management"

Reviewer: ```suggestion
description: "Learn how to use Seqera CLI for bioinformatics workflows, pipeline development, and data management"
```

Use Seqera Platform capabilities to run and manage workflows at scale with AI assistance.

<!-- TODO: Replace with OpenTUI screenshots -->

> ![Use Co-Scientist CLI to debug Platform run errors](./_images/sp-run-debug.gif)

Reviewer: ```suggestion
![Use Seqera CLI to debug Platform run errors](./_images/sp-run-debug.gif)
```

<h2>Learn more</h2>

> - [Co-Scientist CLI](index.md): Co-Scientist CLI overview

Reviewer: ```suggestion
- [Seqera CLI](index.md): Seqera CLI overview
```
